### ISSUE TITLE
More shapes

### ISSUE THREAD

**Issue by low-earth-orbit** [low-earth-orbit/konva-whiteboard#17]
Consider to add more shape options.

- [ ] 5-pt star
- [ ] triangle

**Comment by Maduz0097** [comment]
I have added these new shapes. Can you please assign this to me?

**Comment by low-earth-orbit** [comment]
@Maduz0097 Sure!

Feel free to create a pull request when it's ready. Thanks for your interest in contributing! Let me know if you have any questions.

**Comment by Maduz0097** [comment]
@low-earth-orbit I have opened a pull request, can you please kindly review that? I'm unable to add any reviewers to the pull request thank you.

**Comment by low-earth-orbit** [comment]
@Maduz0097 I've reviewed your PR and merged it. I did found some minor issues related to resizing these shapes; however I did not communicate the specifics regarding the `onTransformEnd` function -- when adding a shape to the canvas, the left top corner of the shape Group is at the mouse down position, and the right bottom coner the mouse up position. I've created a seperate issue for following up.

Thanks for contributing and great work!
